Source/TheLudoGameEngine/Classes/Token.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace TheLudoGameEngine
+{
+    public class Token
+    {
+        public int TokenID { get; set; }
+        public Player Player { get; set; }
+        public int GameBoardPosition { get; set; }
+        public string TokenColor { get; set; }
+        public int TokenNumber { get; set; }
+        public bool InNest { get; set; }
+        public bool InGoal { get; set; }
+
+        public int MoveToken(int dieResult)
+        {
+            for (int i = 1; i <= dieResult; i++)
+            {
+                if (this.GameBoardPosition >= 45)
```

Comment:
Verkar bli ett "off by one" fel här så att man aldrig kan hamna på ruta 45.

Reply:
Tack för input, ja alltså den kedjade for loopen var nå form av animations och stegräknings experimentet. Då en av våra regler är att man behöver få ett jämt för att gå i mål med sin pjäs, annars behöver man backa. Själva metoden räknade stegen per varje runda i loopen(både fram och back) istället för att addera hela tärningens utgång, exempel pjäsen står på pos 43 och slår 5, så skulle den räkna 44, 45, 44, 43, 42. Så varje steg skulle vara nån form av animations motor, därav negationer i MoveToken metoden, men den idén är skrotad och metoden är ändrad till en mer rak uträkning istället för en nästlad loop. Tack för input :)